core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.java

```diff
@@ -301,7 +303,9 @@ public void onDone(String s) {
         @Override
         public void onError(String s) {
           Log.e(TAG_KIWIX, "TextToSpeech Error: " + s);
-          //TODO: Surface to user
+          Toast.makeText(context,
+            R.string.texttospeech_error,
+            Toast.LENGTH_SHORT).show();
```

Comment:
And this?

Reply:
@abdulwd Sure, I'll do that. 
I made it multiple line in order to follow the existing coding style in the file. I'll change the other toasts to follow single-line declaration too.